workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts

```diff
@@ -104,4 +107,29 @@ describe('authentication route handler integration tests', () => {
       expect(data.loggedIn).toBe(false);
     });
   });
+
+  describe('refreshAccessToken', () => {
+    it('should return the id token if the access token is successfully refreshed', async () => {
```

Comment:
Do we need a test for an expired access token?

Reply:
In theory we probably should. In practice, the refresh token is stored in a cookie that expires and deletes at the same time the token itself expires so this is a very edge case. If we want to test it, it will take a decent amount of effort to figure out how to generate a valid but expired refresh token to test with. So I would say handle it in another ticket.